language/en/units.json

```diff
@@ -8,7 +8,10 @@
 		},
 		"dead": "%{name} Wreckage",
 		"heap": "%{name} Debris",
+		"decoycommander": "Decoy",
 		"scavenger": "Scavenger %{name}",
+		"scavcommandernametag": "Scav Commander",
+		"scavdecoycommandernametag": "Scav Decoy",
```

Comment:
Scavengers should always be referred to as Scavengers on the UI, never as Scavs

Reply:
> Scavengers should always be referred to as Scavengers on the UI, never as Scavs

Scav units are all named as "Scav Name" on the UI so this is consistent with that.

Reply:
That has to change

Reply:
why? scav is a far better name

Reply:
The game UI should never present any slang, jargon, acronyms, or abbreviations to the player

Reply:
rename scavs to scavs then